BlazorScheduler/Components/Scheduler.razor.cs

```diff
@@ -211,7 +212,7 @@ public async Task OnMouseUp(int button)
         [JSInvokable]
         public void OnMouseMove(string date)
         {
-            if (_showNewAppointment && _draggingAppointmentAnchor is not null)
+            if (_showNewAppointment && EnableDragging)
             {
                 var day = DateTime.ParseExact(date, "yyyyMMdd", null);
                 (_draggingStart, _draggingEnd) = day < _draggingAppointmentAnchor ? (day, _draggingAppointmentAnchor.Value) : (_draggingAppointmentAnchor.Value, day);
```

Comment:
```suggestion
                var anchor = _draggingAppointmentAnchor!.Value;
                (_draggingStart, _draggingEnd) = day < anchor ? (day, anchor) : (anchor, day);
```

One last thing, seeing a warning here since the IDE can't guarantee that `_draggingAppointmentAnchor` is not null.
This suggestion fixes the warning.